snmp/tests/test_check.py

```diff
@@ -201,6 +201,41 @@ def test_table(aggregator):
     aggregator.all_metrics_asserted()
 
 
+def test_table_regex_match(aggregator):
+    config = [
```

Comment:
To mirror the fact that this contains what would be in the `metrics` field of a profile YAML:

```suggestion
    metrics = [
```